Cubical/Structures/Set/CMon/Bag.agda

```diff
@@ -28,285 +28,63 @@ open import Cubical.Structures.Eq
 open import Cubical.Structures.Arity hiding (_/_)
 open import Cubical.Structures.Set.CMon.QFreeMon
 open import Cubical.Relation.Nullary
+open import Cubical.HITs.PropositionalTruncation as PT
 
 private
   variable
     ℓ : Level
     A : Type ℓ
 
 SymmAction : ∀ {A : Type ℓ} -> Array A -> Array A -> Type ℓ
-SymmAction (n , v) (m , w) = Σ[ σ ∈ Fin n ≃ Fin m ] v ≡ w ∘ σ .fst
+SymmAction (n , v) (m , w) = ∃[ σ ∈ Fin n ≃ Fin m ] v ≡ w ∘ σ .fst
```

Comment:
Maybe using Iso is better here instead of Equiv? The composition etc. is easier.